src/ServiceControl/MessageFailures/Handlers/UnArchiveMessagesByRangeHandler.cs

```diff
@@ -50,6 +51,7 @@ public async Task Handle(UnArchiveMessagesByRange message, IMessageHandlerContex
 
             await domainEvents.Raise(new FailedMessagesUnArchived
             {
+                DocumentIds = patchedDocumentIds.Select(x => x.Document).ToArray(),
```

Comment:
Is this even correct? Aren't we patching against `FailedMessageViewIndex` and that index is using `FailedMessage` where the `Id` is determined by

```
        public static string MakeDocumentId(string messageUniqueId)
        {
            return $"{CollectionName}/{messageUniqueId}";
        }
```

which then would mean we'd have to do something like


```
DocumentIds = patchedDocumentIds.Select(x => FailedMessage.GetMessageIdFromDocumentId(x.Document)).ToArray(),
```

?